src/Components/Officers.js

```
@@ -3,8 +3,46 @@ import "../CSS/Officers.css";
 import ProfileCard from "./profileCard";
 
 const Officers = () => {
-  const [selectedYear, setSelectedYear] = useState(new Date().getFullYear());
+  
+  const [selectedYear, setSelectedYear] = useState(null);
   const [teamMembers, setTeamMembers] = useState([]);
+  const [yearOptions, setYearOptions] = useState([]);
+
+  useEffect(() => {
+    const yearSetup = async () => {
+      const currentDate = new Date();
+      const availableYears = [];
+
+      // officer data will be out at the end of June, this will update beginning of July
+      const defaultToThisYear = currentDate.getMonth() > 0; 
+
+      // check if officer json data for the current year exists first
+      if (defaultToThisYear) {
+        try {
+          await import(`../Data/officers${currentDate.getFullYear()}.json`);
+          availableYears.push(currentDate.getFullYear());
+        } catch (error) {
+          // officer data for this year won't be added
```

Maybe add a log or remove the catch entirely? I'm not sure if it's a good idea to just have an empty catch...